plugins/inventory/sops.py

```diff
@@ -0,0 +1,56 @@
+import os
+from tempfile import NamedTemporaryFile
+
+from ansible.errors import AnsibleParserError
+from ansible.plugins.inventory import BaseInventoryPlugin
+from ansible.plugins.loader import inventory_loader
+
+
+from ansible_collections.community.sops.plugins.module_utils.sops import Sops
+
+
```

Comment:
The plugin is missing `DOCUMENTATION`, which every plugin (except jinja2 test/filter plugins) **must** have.

Reply:
Hey there @felixfontein that is what I asked on the PR intro, should I write the docs referring people to yaml's inventory or should I come up with something else? Any tips are welcome

Reply:
Hello @jaysonsantos, welcome!

You can take inspiration from other inventories plugin (like [AWS EC2 plugin](https://github.com/ansible-collections/amazon.aws/blob/main/plugins/inventory/aws_ec2.py#L7)

Please also add an `EXAMPLES` section (see [here](https://github.com/ansible-collections/amazon.aws/blob/main/plugins/inventory/aws_ec2.py#L104) for inspiration).